libmamba/src/core/curl.cpp

```diff
@@ -446,6 +446,21 @@ namespace mamba
         return *this;
     }
 
+    CURLHandle& CURLHandle::set_url(const std::string& url, const proxy_map_type& proxies)
```

Comment:
This should not be required, ot deserves a better name if we need to reset the URL of an existing handler.